tests/components/konnected/test_config_flow.py

```diff
@@ -0,0 +1,689 @@
+"""Tests for Konnected Alarm Panel config flow."""
+from unittest.mock import patch
+
+from homeassistant.components.konnected import config_flow
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_SSDP
+
+from tests.common import MockConfigEntry, mock_coro
+
+
+async def test_flow_works(hass, aioclient_mock):
+    """Test config flow ."""
+    flow = config_flow.KonnectedFlowHandler()
```

Comment:
Don't create the flow directly. Use the core interfaces. See the hue config flow tests for good examples.

Reply:
👍